### DISCUSSION TITLE
[✨]Automatically compress images to support more devices.

### DISCUSSION THREAD

**Discussion by xhcy1314** [QwikDev/qwik-evolution#115] [Proposals For Qwik]
### Is your feature request related to a problem?

Currently, image transcoding in the project only supports a single configuration, allowing only AVIF, WebP, or PNG to be passed in. Additionally, the transcoded resources do not include the original image format. I believe this approach does not align with a popular framework because a single configuration can lead to situations where devices that do not support a certain image format cannot view the images, causing anomalies.

### Describe the solution you'd like

Compressing and transcoding while retaining the original image format, adding AVIF, WebP, and the original image format allows for automatic selection of the optimal image based on device capabilities.
PR: https://github.com/QwikDev/qwik/pull/6161

### Describe alternatives you've considered

-

**Comment by gioboa** [comment]
We moved this issue to `qwik-evolution` repo to create a RFC discussion for this.
Here is our [Qwik RFC process](https://github.com/QwikDev/qwik-evolution?tab=readme-ov-file#qwik-rfc-process) thanks.
